.travis.yml

```diff
@@ -1,13 +1,19 @@
 jobs:
   include:
-    - name: "18.04 pycodestyle and native run (py3.6)"
+    - name: "20.04 flake8 and native run (py3.8)"
       os: linux
-      dist: bionic
+      dist: focal
       language: shell
-      install: sudo apt-get update; sudo apt install pycodestyle python3-pexpect;
+      install: sudo apt-get update; sudo apt install flake8 python3-pexpect;
       script:
-        - "pycodestyle sos"
+        - "flake8 sos tests bin/*"
```

Comment:
Is `flake8` a complete replacement for `pycodestyle`, as in it checks _all_ the same linter tests and then some? It's been a while since I've looked into it and I can't rightly remember if it's a complete replacement or one that is best used in tandem.

I have no objections to changing to `flake8`, but I just want to be sure we won't be missing something from `pycodestyle`.

Reply:
flake8 includes pycodestyle itself, so it should be equivalent unless ignores are different.  Double checked,

pycodestyle ignores E121,E123,E126,E226,E24,E704,W503,W504
flake8 ignore 'E126', 'E24', 'E226', 'E121', 'E704', 'E123', 'W504', 'W503']

So flake8 seems to base it's ignore list on pycodestyles.